src/main/java/seedu/address/logic/commands/visit/AddVisitCommand.java

```diff
@@ -23,16 +23,24 @@
 
     public static final String COMMAND_WORD = "addVisit";
 
-
     public static final String MESSAGE_SUCCESS = "New visit added: %1$s";
-    public static final String MESSAGE_DUPLICATE_VISIT = "This visit already exists in the visit book";
-
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Add a new visit to the visit book "
+    public static final String MESSAGE_DUPLICATE_VISIT = "This visit already exists in the address book";
+    public static final String MESSAGE_NO_WARNING = MESSAGE_SUCCESS;
+    public static final String MESSAGE_INFECTED_MADE_VISIT = MESSAGE_SUCCESS + "\n"
+            + "The following person is infected and "
+            + "may have violated the Stay-Home Notice.";
+    public static final String MESSAGE_QUARANTINED_MADE_VISIT = MESSAGE_SUCCESS + "\n"
+            + "The following person is in quarantine and "
+            + "may have violated the Stay-Home Notice.";
+    public static final String MESSAGE_INFECTED_AND_QUARANTINED_MADE_VISIT = MESSAGE_SUCCESS + "\n"
+            + "The following person is infected and "
+            + "is in quarantine. The Stay-Home Notice may have been violated.";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Add a new visit to the visits list "
             + "by the personId of visit, locationId of visit and date of visit "
             + "Existing visits list will be updated.\n"
-            + "Parameters: PersonId (must be a positive integer within the range of person book) ,"
-            + "LocationId (must be a positive integer within the range of location book),  "
-            + "Date (must be in the format of yyyy-MM-dd "
+            + "Parameters: PersonId (must be a positive integer within the range of person book),\n"
+            + "LocationId (must be a positive integer within the range of location book),\n"
```

Comment:
Please change this to index instead of id. Only index number is exposed to the user.

Reply:
Done